notebooks/source/conf.py

```diff
@@ -109,7 +109,7 @@
 # documentation.
 #
 html_theme_options = {
-    'logo_only': True
+    # 'logo_only': True
```

Comment:
I removed this since without `NumPyro Tutorials` in the nav bar, the Pyro logo looks confusing with a version number of `0.2.0` for NumPyro. Ideally `NumPyro` in the font below the logo and above the version would be much cleaner, but that can wait for later. I also am not sure if there's a way to do this in conf.